### PR TITLE
DCOS-40413: Only show Table column shadow on scroll

### PR DESCRIPTION
**Pull request by mperrotti**
shadowscroll_after

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -80,6 +80,17 @@ pipeline {
       }
```
Comment (weblancaster) [inline Jenkinsfile]: I believe this should not be in your PR.

File: .releaserc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (weblancaster) [inline .releaserc]: I believe this should not be in your PR

File: README.md [hunk 3]
```diff
@@ -72,9 +72,9 @@ A recommended reading is [Better Specs](http://www.betterspecs.org/), we put
 [real effort](https://github.com/dcos/dcos-ui/pull/2524) in making sure we
```
Comment (weblancaster) [inline README.md]: I believe this should not be in your PR

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dcos/ui-kit",
-  "version": "1.3.0",
+  "version": "0.0.0-dev+semantically-released",
```
Comment (weblancaster) [inline package.json]: why was this changed?

File: .releaserc (hunk 2 above)
Comment (nLight) [reply .releaserc]: I agree with Michael, let's please keep those separate

File: package.json (hunk 4 above)
Comment (Poltergeist) [reply package.json]: yes

**Comment by nLight** [comment]
flicts

File: package.json (hunk 4 above)
Comment (Poltergeist) [reply package.json]: https://github.com/dcos-labs/ui-kit/pull/198

**Comment by juliangieseke** [comment]
@mperrotti this branch is messed up, you own commit is in here twice 🤔

**Comment by nLight** [comment]
flicts

**Comment by Poltergeist** [comment]
I will rebase this Pr and solve the conflict to move this forward.

**Comment by Poltergeist** [comment]
I was not able to rebase this and fix the conflicts, unfortunately.

**Comment by Poltergeist** [comment]
I thought we only want to show the shadow while scrolling, although I like how the functionality is working.

**Comment by mperrotti** [comment]
@Poltergeist - I've always seen it done this way, where the shadow remains visible as long as the user is scrolled over. @leemunroe or @lilysamimi - what do you think?

**Comment by lilysamimi** [comment]
@mperrotti It's perfect! 🙇🏻‍♀️

@leemunroe check out the latest shadow improvements! https://cl.ly/0i3x1i0S0X1o

One minor thing that's not related to this PR and I'm not sure if it's a bug but I want to call it out just in case ... the "Zip Code" header is not aligned with the rest of the column, there is a column with no header but the table cells are called "empty", and the "City" column is being treated as a primary column it seems as the table cells are in bold.
